content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -362,6 +362,41 @@ For more information on inter-pod affinity/anti-affinity, see the
 You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
 as well, which allow a *node* to *repel* a set of pods.
 
+## nodeName
+
+`nodeName` is the simplest form of node selection constraint, but due
+to its limitations its use is _not_ recommended.  `nodeName` is a
+field of PodSpec.  If it is non-empty, the scheduler ignores the pod
+and the kubelet running on the named node tries to run the pod.  Thus,
+if `nodeName` is provided in the PodSpec, it takes precedence over the
+above methods for node selection.
+
+Some of the limitations of using `nodeName` to select nodes are:
+
+-   If the named node does not exist, the pod will be deleted after a
```

Comment:
Everything looks good, but I am not sure about this point. Did you find it in any of our documentations?

Reply:
I tested it myself against a cluster running Kubernetes 1.10. The pod existed briefly with a pending status and then was deleted. I did the same in a job PodSpec and the job kept creating pods only to have them go from pending to deleted, until I deleted the job.

Reply:
I tested the insufficient resource cases similarly.

Reply:
@bsalamat any thoughts on who the proper technical reviewer should be for this?

Reply:
that behavior is part of the PodGC controller: https://github.com/kubernetes/kubernetes/blob/3be5e73eff27869b5da71cacc1c364e8000f8cac/pkg/controller/podgc/gc_controller.go#L142-L169

That controller running is not an API guarantee, and I don't see it exercised in a conformance test.

Reply:
I've changed this to "If the named node does not exist, the pod will not be created."